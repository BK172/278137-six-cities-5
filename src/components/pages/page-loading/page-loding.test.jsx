import React from "react";
import renderer from "react-test-renderer";
import PageLoading from "./page-loading";

describe(`PageLoading component test`, () => {
  it(`Should render PageLoading correctly`, () => {
    const tree = renderer
      .create(
          <PageLoading />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
