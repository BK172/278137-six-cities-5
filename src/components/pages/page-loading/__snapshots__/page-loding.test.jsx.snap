// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLoading component test Should render PageLoading correctly 1`] = `
<div
  className="container"
>
  <h1>
    Loading...
  </h1>
</div>
`;
